src/modules/auth/index.js

```diff
@@ -280,14 +280,27 @@ export default class AuthModule extends Module {
 			if (that.App.Store != null) {
 				that.App.Store.dispatch({ type: types.AUTH_SESSION_EXPIRATION, sessionExpired: true });
 				// Disable buttons and pagination in the whole screen
-				[...document.querySelectorAll('[class^="btn"]:not(.alert-button), [class*=" btn"]:not(.alert-button), .page-item')].forEach(i => {
+				[...document.querySelectorAll('[class^="btn"]:not(.alert-button), [class*=" btn"]:not(.alert-button), .btn-group a, .page-item, input, select')].forEach(i => {
 					i.classList.add("disabled");
 					i.setAttribute("disabled", "");
 				});
 				// Disable link without class "nav-link" in the whole screen
 				[...document.querySelectorAll('a:not(.nav-link)')].forEach(i => {
 					i.classList.add("disabled-link");
 				});
+				// Disabling the ability to click on parts of widgets
+				[...document.querySelectorAll('.recharts-surface')].forEach(i => {
+					i.style.pointerEvents = "none"
+				});
+				// Disabling the ability to click on item in the Library
+				[...document.querySelectorAll('.tree-menu-item')].forEach(i => {
```

Comment:
@aringocode not sure about this, since what if we decide to change the library of tree menu?